mapper/tests.py

```diff
@@ -323,7 +326,19 @@ def test_form_correct_submission(self):
         self.assertRedirects(response, reverse('migration-list'))
 
         [migration] = MigrateSubscription.objects.all()
+        # Test object is created correctly
         self.assertEqual(migration.table_name, 'testtable1')
         self.assertEqual(migration.column_name, tables['testtable1'][0])
         self.assertEqual(migration.from_messageset, 1)
         self.assertEqual(migration.to_messageset, 2)
+
+        # Test object history is created correctly
+        [history] = LogEntry.objects.filter(object_id=migration.pk)
```

Comment:
My Python's still a bit rusty so I haven't seen this notation for a while. Am I right in thinking that this will assign `history` to the one and only object returned by `LogEntry.objects.filter(object_id=migration.pk)` and will error if that query returns anything other than a list with one element?